.travis.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       dist: bionic
       language: python
       python: "3.6"
-      install: pip install -r requirements.txt; python3 setup.py install;
+      install: pip install -r test-requirements.txt; python3 setup.py install;
```

Comment:
I don't see why we'd need to split the requirements into 2 files. Not least of which, this stops pulling in pexpect which once we have testing for `collect` figured out, we'd need to add it to the new file as well. I don't see a benefit in splitting out the requirements for a single different module to install.

Reply:
I am still experimenting here. The main reason I split the requirements though is that packages such as `pycodestyle` are not necessary for `sos` and are only needed for testing (unless I am missing something).